search/src/Components/upload/uploader_voc.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import 'react-dropzone-uploader/dist/styles.css';
+import Dropzone from 'react-dropzone-uploader';
+import {jsonSchemaTitle} from '../error/utils';
+import axios from 'axios';
+import Cookies from 'js-cookie';
+
+
+const tmp_voc = Math.random().toString(36).slice(-8);
```

Comment:
This is a global constant, and could be the same for every upload if the user does multiple in one session. Maybe we should set this inside the upload stepper?

Reply:
This works as expected. If you put it inside, it applies to single selection of files, say if you upload one voc and decide to upload another one, they will go to different folder. Not sure why the js scope is working like this, but I came up with this solution by practice.

Reply:
> If you put it inside, it applies to single selection of files

That's why I suggested initialising this as part of the stepper, not part of the hook. It's a unique token per upload, so that's context that can be maintained by the stepper.

Reply:
I put it in the uploader state now